contracts/lib/AddressToAddressWhiteList.sol

```diff
@@ -37,19 +37,19 @@ contract AddressToAddressWhiteList is
 
     /* ============ State Variables ============ */
 
-    address[] public addresses;
-    mapping(address => address) public addressToAddressWhiteList;
+    address[] public keys;
+    mapping(address => address) public keysToValues;
```

Comment:
Looking at this again, we can just call it `whitelist` instead of `keysToValues` lol.

Reply:
+1